numpy/linalg/linalg.py

```diff
@@ -2357,14 +2357,13 @@ def multi_dot(arrays): return functools.reduce(np.dot, arrays)
     >>> # or
     >>> A.dot(B).dot(C).dot(D)
 
-
-    Example: multiplication costs of different parenthesizations
-    ------------------------------------------------------------
-
+    Notes
```

Comment:
We already have a notes section higher up.

Also, the changes to this file kinda feel different to all the rest - maybe put them in their own PR?